extensions/wikia/ArticleVideo/ArticleVideoController.class.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+class ArticleVideoController extends WikiaController {
+	public function index() {
+		$wg = F::app()->wg;
+
+		$articleId = RequestContext::getMain()->getTitle()->getArticleID();
+		$thumbnailPath = $wg->videoMVPArticles[$wg->cityId][$articleId]['thumbnailPath'];
+
+		$this->setVal( 'thumbnailUrl', $wg->extensionsPath . $thumbnailPath );
+		$this->setVal( 'closeIconUrl', $wg->extensionsPath . '/wikia/ArticleVideo/images/close.svg' );
```

Comment:
Can we talk with Design Team about it? We shouldn't introducing multiple assets of the same type.